python-avd/pyavd/_anta/input_factories/connectivity.py

```diff
@@ -46,24 +46,28 @@ def create(self) -> list[VerifyLLDPNeighbors.Input] | None:
                 self.logger_adapter.debug(LogMessage.INTERFACE_SHUTDOWN, interface=intf.name)
                 continue
 
-            if not intf.peer or not intf.peer_interface:
+            if not intf.metadata.peer or not intf.metadata.peer_interface:
                 self.logger_adapter.debug(LogMessage.INPUT_MISSING_FIELDS, identity=intf.name, fields="peer, peer_interface")
```

Comment:
Should logs include new/full path as well?

```suggestion
                self.logger_adapter.debug(LogMessage.INPUT_MISSING_FIELDS, identity=intf.name, fields="metadata.peer, metadata.peer_interface")
```

Reply:
updated